scripts/trelloOnboardingBuilder.js

```diff
@@ -55,6 +72,17 @@ var app = new Vue({
             this.message = "Reorganizing Learning Path...";
             const { idList: learningPathListId } = await getCard(this.learningPathCardId, { trelloApiKey: this.trelloApiKey, trelloOAuth1: this.trelloOAuth1 });
             const allCardsInLearningPath = await fetchCardsInList(learningPathListId, { trelloApiKey: this.trelloApiKey, trelloOAuth1: this.trelloOAuth1 });
+            const initialTimelineCardPosition = computeInitialTimelineCardPositions();
+
+            function computeInitialTimelineCardPositions() {
+                const timelineCardPosition = {};
+                for (const card of allCardsInLearningPath) {
+                    if (! Object.keys(timelineInformationMapping).includes(card.name)) { continue; }
```

Comment:
@AdriAt360 Try never using continue / break in loops when possible
If you want function guard clause absolutely you can use `Array.reduce`
Or this whole silly algo could go to a `allCardsInLearningPath.filter().map()` ideally which will make it fast tor understand :wink:

Reply:
@pp0rtal I partially agree ^^
- I'm not fond of the reduce syntax (I feel it's less readable)
- the `filter.map` is another option, but I feel the readability is quite similar in real life
- and in the other use of `for` I can also simply `await` things, that I would not be able to do with `map`

Reply:
@AdriAt360 I agree for promises (while it's not the case in this specific code)
I used for year https://www.npmjs.com/package/bluebird for complex algorithms, which is great so you can use filter/map/each/reduce with promises :heart_eyes: